app/models/vaccine.rb

```diff
@@ -1,7 +1,26 @@
 # frozen_string_literal: true
 
 class Vaccine < ApplicationRecord
+  CVX = 'http://hl7.org/fhir/sid/cvx'
+
   has_many :immunizations, dependent: :restrict_with_exception
 
   default_scope { order(:name) }
+
+  def self.seed
```

Comment:
I wouldn't change how the seeding works since it's being worked on in another branch.

Reply:
I needed to seed Vaccine for the test code; since the tests are run in parallel I end up seeding multiple times and decided to put it into one function. I can't use test fixtures because YAML parses the CVX code as an Integer, while its suppose to be a string.